### PR TITLE
Ember cli 3 28

### PR DESCRIPTION
**Pull request by ctjhoa**

### CODE REVIEW

File: tests/integration/components/aria-tab-list-test.js [hunk 1]
```diff
@@ -12,30 +12,30 @@ module('Integration | Component | aria-tab-list', function (hooks) {
 
     await render(hbs`<AriaTabList />`);
 
-    let tabList = this.element.querySelector('[role="tablist"]');
-    assert.equal(tabList.textContent.trim(), '');
+    assert.dom('[role="tablist"]').hasText('');
```
Comment (frykten) [inline tests/integration/components/aria-tab-list-test.js:15]: `.hasNoText()`
non ?

File: tests/integration/components/aria-tab-panel-test.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module('Integration | Component | aria-tab-panel', function (hooks) {
   test('it renders with sane defaults', async function (assert) {
     await render(hbs`<AriaTabPanel>Hola</AriaTabPanel>`);
 
-    assert.equal(this.element.textContent.trim(), '');
+    assert.dom(this.element).hasText('');
```
Comment (frykten) [inline tests/integration/components/aria-tab-panel-test.js:12]: Même question pour chacune de ces assertions sur une chaîne vide

File: tests/integration/components/aria-tabs-test.js [hunk 3]
```diff
@@ -51,15 +51,16 @@ module('Integration | Component | aria-tabs', function (hooks) {
     assert.equal(tabs.length, 4);
     assert.equal(tabPanels.length, 4);
     assert.equal(disabledTab.length, 1);
```
Comment (frykten) [inline tests/integration/components/aria-tabs-test.js:53]: https://github.com/simplabs/qunit-dom/blob/master/API.md#exists
Tu peux lui passer le nombre désiré

Comment (frykten) [reply tests/integration/components/aria-tabs-test.js:53]: Et du coup, même réflexion à chaque `assert.equal(list.length, N)`

File: config/ember-try.js [hunk 4]
```diff
@@ -6,27 +6,19 @@ const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
...
       },
       {
         name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
-            'ember-source': '~3.24.5',
+            'ember-source': '~3.24.3',
           },
         },
       },
```
Comment (mayatron) [inline config/ember-try.js:24]: Recommend adding scenario for `ember-lts-3.28`.

**Review by mayatron — COMMENTED** [review]
Looking forward to this to be merged, thank you! :)

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -57,7 +57,6 @@ jobs:
       fail-fast: true
       matrix:
         ember-try-scenario:
-          - ember-lts-3.16
           - ember-lts-3.20
           - ember-lts-3.24
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply .github/workflows/main.yml:61]: As project will rely on [ember-source 3.28](https://github.com/concordnow/ember-aria-tabs/pull/335/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R65) it's not useful to add an ember try scenario. `ember test` will already test 3.28

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -57,7 +57,6 @@ jobs:
...
           - ember-release
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply .github/workflows/main.yml:62]: As project will rely on [ember-source 3.28](https://github.com/concordnow/ember-aria-tabs/pull/335/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R65) it's not useful to add an ember try scenario. `ember test` will already test 3.28

File: config/ember-try.js (hunk 4 above)
Comment (ctjhoa) [reply config/ember-try.js:24]: As project will rely on [ember-source 3.28](https://github.com/concordnow/ember-aria-tabs/pull/335/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R65) it's not useful to add an ember try scenario. `ember test` will already test 3.28

File: tests/integration/components/aria-tab-list-test.js (hunk 1 above)
Comment (ctjhoa) [reply tests/integration/components/aria-tab-list-test.js]: 52028481b97675ad06375c89908bd5c87f5d70a0

File: tests/integration/components/aria-tab-panel-test.js (hunk 2 above)
Comment (ctjhoa) [reply tests/integration/components/aria-tab-panel-test.js]: 52028481b97675ad06375c89908bd5c87f5d70a0

File: tests/integration/components/aria-tabs-test.js (hunk 3 above)
Comment (ctjhoa) [reply tests/integration/components/aria-tabs-test.js:53]: Not relevant here as we need to tests individual elements in following lines

**Pull request merged**